index.d.ts

```diff
@@ -21,7 +21,7 @@ declare module "express-validator" {
    */
   function ExpressValidator(options?: ExpressValidator.Options.ExpressValidatorOptions): express.RequestHandler;
 
-  export = ExpressValidator;
+  export default ExpressValidator;
```

Comment:
The way the lib is [imported](https://github.com/ctavan/express-validator/blob/841944e40b5bbc48aa5a92287ccb8436b26e9e5c/test/typeDefinitionTest.ts#L1) in the test is "generic": It works for all sorts of modules.

But if you try to do it the ES6 way `import ev from 'express-validator'` it would fails, even if you `export default ExpressValidator`